### PR TITLE
button block lable for landing page in all Translation language folders

### PR DESCRIPTION
**Pull request by disha1611**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.

### CODE REVIEW

File: plugins/GrapesJsBuilderBundle/Translations/de/javascript.ini [hunk 1]
```diff
@@ -8,3 +8,4 @@ grapesjsbuilder.dynamicContentBlockLabel="Dynamischer Inhalt"
 grapesjsbuilder.dynamicContentBtnLabel="Speichern"
 grapesjsbuilder.dynamicContentModalTitle="Dynamischen Inhalt bearbeiten"
 grapesjsbuilder.assetManager.noAssets="Keine <b>Assets</b> hier, zum Hochladen ziehen"
+grapesjsbuilder.buttonBlockLabel="Taste"
```
Comment (adiux) [inline plugins/GrapesJsBuilderBundle/Translations/de/javascript.ini]: Taste is a correct translation, but not a very common one in this scenario. I checked how it was translated elsewhere. E.g. with the Forms we use the translation "Button" (mautic.form.field.type.button="Button"). The same as in English.

File: plugins/GrapesJsBuilderBundle/Translations/cs/javascript.ini [hunk 2]
```diff
@@ -8,3 +8,4 @@ grapesjsbuilder.dynamicContentBlockLabel="Dynamický obsah"
 grapesjsbuilder.dynamicContentBtnLabel="Uložit"
 grapesjsbuilder.dynamicContentModalTitle="Upravit dynamický obsah"
 grapesjsbuilder.assetManager.noAssets="Nejsou zde žádné <b>assety</b>, přetáhněte sem soubory pro nahrání."
+grapesjsbuilder.buttonBlockLabel="Knoflík"
```
Comment (kuzmany) [inline plugins/GrapesJsBuilderBundle/Translations/cs/javascript.ini]: I am not sure If this is regular Czech translation :) @escopecz

Comment (disha1611) [reply plugins/GrapesJsBuilderBundle/Translations/cs/javascript.ini]: I have found tlačítka on google translate for string buttons but when I translate for string button it gives Knoflík. So guys if you have idea about what correct translation is for the above language please let me know. I will change and commit the file again. Thanks @kuzmany @escopecz

Comment (escopecz) [reply plugins/GrapesJsBuilderBundle/Translations/cs/javascript.ini]:
```suggestion
grapesjsbuilder.buttonBlockLabel="Tlačítko"
```
This is much better. "Knoflík" is used to button up a shirt.

**Comment by RCheesley** [comment]
@kuzmany @adiux are we good to go with the suggested changes being made?

**Comment by RCheesley** [comment]
Safe to ignore the Codecov as this is not touching any PHP files.

**Review by RCheesley — APPROVED** [review]

**Comment by RCheesley** [comment]
@all-contributors please add @disha-pishavadia24 for code

**Comment by RCheesley** [comment]
@all-contributors please add @adiux for review

**Pull request merged**